geopandas/geodataframe.py

```diff
@@ -88,6 +88,8 @@ def __init__(self, *args, **kwargs):
 
         if isinstance(arg, (dict, list, pd.Series, np.ndarray)):
             arg = pd.DataFrame(arg, **kwargs)
+        elif arg is None:
+            arg = pd.DataFrame(**kwargs)
```

Comment:
this should not be necessary, as `DataFrame(None)` (what you would have in `pd.DataFrame(arg, **kwargs)`) works

Reply:
but you probably run into problems later on with eg `BlockManager(blocks, [columns, index])` where the index does not match the number of elements in the geometry column